RELEASE_NOTES.md

```diff
@@ -168,7 +168,7 @@ Chef now ships with Ruby 2.6.3. This new version of Ruby improves performance an
 
 Platform and platform_family detection on Linux has been rewritten to utilize the latest config files on modern Linux distributions before falling back to slower and fragile legacy detection methods. Ohai will now begin by parsing the contents of `/etc/os-release` for OS information if available. This feature improves the reliability of detection on modern distros and allows detection of new distros as they are released.
```

Comment:
For formatting purposes, does "Platform" or "platform_family" need some `type` of formatting in this paragraph? I wasn't sure if we wanted to include certain annotation or not.

Reply:
These should probably get formatting

Reply:
All, some, or certain instances?

Reply:
Those are node attributes not generic references to a platform. We're really talking about node['platform'] and node['platform_family'] so they should get the special treatment.